modules/core/src/scenegraph/gltf-instantiator/gltf-material.js

```diff
@@ -75,25 +75,16 @@ class GLTFEnv {
   }
 
   makeCube(id, func) {
-    return Promise.all([
-      func('right'),
-      func('top'),
-      func('front'),
-      func('left'),
-      func('bottom'),
-      func('back')
-    ]).then(faces => {
-      return new TextureCube(this.gl, {
-        id,
-        pixels: {
-          [this.gl.TEXTURE_CUBE_MAP_POSITIVE_X]: faces[0],
-          [this.gl.TEXTURE_CUBE_MAP_POSITIVE_Y]: faces[1],
-          [this.gl.TEXTURE_CUBE_MAP_POSITIVE_Z]: faces[2],
-          [this.gl.TEXTURE_CUBE_MAP_NEGATIVE_X]: faces[3],
-          [this.gl.TEXTURE_CUBE_MAP_NEGATIVE_Y]: faces[4],
-          [this.gl.TEXTURE_CUBE_MAP_NEGATIVE_Z]: faces[5]
-        }
-      });
+    return new TextureCube(this.gl, {
+      id,
+      pixels: {
+        [this.gl.TEXTURE_CUBE_MAP_POSITIVE_X]: func('right'),
```

Comment:
Are these names part of the standard in any way? (right, left, ...)? 

Or just an artifact of the examples we happen to use?

If not, can we send in something more generic? E.g. the WebGL constant?

Reply:
+1

Also, nit: I generally prefer ordering sequential constants in the order of the values:
<img width="533" alt="screen shot 2019-02-25 at 11 24 19 am" src="https://user-images.githubusercontent.com/476017/53351966-fc22f780-38ef-11e9-8d49-ce304cc43a84.png">
Makes it easier to switch to an array if that ever happens.